### PR TITLE
Added json parameters to windows ARM template

### PR DESCRIPTION
**Pull request by NimishaS**

### CODE REVIEW

**Comment by azurecla** [comment]
Hi __@NimishaS__, I'm your friendly neighborhood Azure Pull Request Bot (You can call me AZPRBOT). Thanks for your contribution!
    <span>You've already signed the contribution license agreement. Thanks!</span>
        <span>We will now validate the agreement and then real humans will evaluate your PR.</span>

TTYL, AZPRBOT;

File: chef-extension-windows-vm/azuredeploy.json [hunk 1]
```diff
@@ -227,6 +227,12 @@
         "type": "ChefClient",
         "typeHandlerVersion": "1210.12",
         "settings": {
+          "bootstrap_options":
```
Comment (siddheshwar-more) [inline chef-extension-windows-vm/azuredeploy.json]: @NimishaS 
Please update https://github.com/Azure/azure-quickstart-templates/blob/master/chef-extension-windows-vm/azuredeploy-parameters.json for `bootstrap_options`

**Comment by singhkay** [comment]
@NimishaS Can you make sure all "description" tags are lower case in the main template file. Thx!

**Comment by NimishaS** [comment]
@singhkay , updated the "description" tag.

**Comment by singhkay** [comment]
@NimishaS According to last error from bot looks like there is an extra parameter in the parameters file "vmExtensionName" Can you check?

**Comment by NimishaS** [comment]
@singhkay, done!

**Comment by singhkay** [comment]
Thanks! @NimishaS 

merged

**Pull request merged**